src/hotspot/share/oops/oop.inline.hpp

```diff
@@ -25,7 +25,6 @@
 #ifndef SHARE_OOPS_OOP_INLINE_HPP
 #define SHARE_OOPS_OOP_INLINE_HPP
 
-#include "gc/shared/collectedHeap.hpp"
```

Comment:
`utilities/globalDefinitions.hpp` for `HeapWord` is missing.
`globals.hpp` for some globals.
`oopsHierarchy.hpp` for `narrowKlass`
`utilties/debug.hpp` for `assert`

Reply:
Fixed. Thanks for the review.